src/bundle/wysiwyg-preset.ts

```diff
@@ -64,7 +65,10 @@ export const BundlePreset: ExtensionAuto<BundlePresetOptions> = (builder, opts)
             paragraphKey: f.toPM(A.Text),
             paragraphPlaceholder: (node: Node, parent?: Node | null) => {
                 const isDocEmpty =
-                    !node.text && parent?.type.name === BaseNode.Doc && parent.childCount === 1;
+                    !node.text &&
+                    ((opts.emptyRowPlaceholder && parent?.type.name === 'doc') ||
```

Comment:
`opts.emptyRowPlaceholder ` 
It seems that the logic of the behavior of an empty line for a paragraph and for a dock should behave differently, I suggest creating 2 variables